runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -168,8 +166,8 @@ protected void startUp() throws Exception
 		overlayManager.add(targetMinimapOverlay);
 
 		if (client.getGameState() == GameState.LOGGED_IN
-			&& config.amount() != -1
-			&& !config.taskName().isEmpty())
+				&& config.amount() != -1
+				&& !config.taskName().isEmpty())
```

Comment:
Don't change whitespacing unless the code itself was also changed. (even though this style is preferred)